### PR TITLE
The 1st batch patch for msdn url 

### PR DESCRIPTION
**Pull request by luyajun0205**
The 1st batch(100 files) patch for msdn url

### CODE REVIEW

**Comment by luyajun0205** [comment]
@BillWagner @mairaw @TianqiZhang This is first 100 files change, please help to review.

File: xml/ns-System.Linq.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   
  The <xref:System.Linq.Queryable> class contains [!INCLUDE[vbteclinq](~/includes/vbteclinq-md.md)] standard query operators that operate on objects that implement <xref:System.Linq.IQueryable%601>.  
   
- For information about LINQ to SQL, see [LINQ to SQL &#91;LINQ to SQL&#93;](https://msdn.microsoft.com/library/73d13345-eece-471a-af40-4cc7a2f11655).  
+ For information about LINQ to SQL, see [LINQ to SQL &#91;LINQ to SQL&#93;](/dotnet/framework/data/adonet/sql/linq/).
```
Comment (mairaw) [inline xml/ns-System.Linq.xml:14]:
```suggestion
 For more information, see [LINQ to SQL](/dotnet/framework/data/adonet/sql/linq/).  
```

File: xml/ns-System.Printing.Interop.xml [hunk 2]
```diff
@@ -11,7 +11,7 @@
     </remarks>
     <altmember cref="N:System.Printing" />
     <altmember cref="T:System.Windows.Controls.PrintDialog" />
-    <related type="ExternalDocumentation" href="https://msdn.microsoft.com/library/windows/desktop/ff686805(v=vs.85).aspx">Printing and</related>
-    <related type="ExternalDocumentation" href="https://msdn.microsoft.com/library/windows/desktop/dd162871(v=vs.85).aspx">Print Spooler</related>
+    <related type="ExternalDocumentation" href="/windows/win32/printdocs/printdocs-printing(v=vs.85).aspx">Printing and</related>
```
Comment (mairaw) [inline xml/ns-System.Printing.Interop.xml:14]:
```suggestion
    <related type="ExternalDocumentation" href="/windows/win32/printdocs/printdocs-printing">Printing and</related>
```

File: xml/ns-System.Printing.Interop.xml [hunk 3]
```diff
@@ -11,7 +11,7 @@
...
+    <related type="ExternalDocumentation" href="/windows/win32/printdocs/print-spooler(v=vs.85).aspx">Print Spooler</related>
```
Comment (mairaw) [inline xml/ns-System.Printing.Interop.xml:15]:
```suggestion
    <related type="ExternalDocumentation" href="/windows/win32/printdocs/print-spooler">Print Spooler</related>
```

File: xml/ns-System.EnterpriseServices.xml [hunk 4]
```diff
@@ -5,7 +5,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- For information about using <xref:System.EnterpriseServices>, see [Writing Serviced Components](https://msdn.microsoft.com/library/f109ee24-81ad-4d99-9892-51ac6f34978c)  
+ For information about using <xref:System.EnterpriseServices>, see [Writing Serviced Components](https://www.microsoft.com/en-us/download/details.aspx?id=55984)
```
[the remark this answers is not in the document]
Comment (luyajun0205) [reply xml/ns-System.EnterpriseServices.xml:8]: So, i should let what it was be, no update here?

[the remark this answers is not in the document]
Comment (luyajun0205) [reply xml/ns-System.EnterpriseServices.xml]: I have updated to value '<remarks>To be added.</remarks>'

**Comment by TianqiZhang** [comment]
all the changes are addressed now, @mairaw could you please review again?

**Pull request merged**